bubble_sort.c

```diff
@@ -11,16 +11,27 @@ int main()
 		scanf("%i",&arr[i]);
 
 	}
-	for(int i=0;i<length;++i)
-	{
-		for(int j=0;j<length-i-1;++j)
-		{
-			if(arr[j]>arr[j+1])
-			{
-				int temp;
-				temp=arr[j];
-				arr[j]=arr[j+1];
-				arr[j+1]=temp;
+// 	for(int i=0;i<length;++i)
+// 	{
+// 		for(int j=0;j<length-i-1;++j)
+// 		{
+// 			if(arr[j]>arr[j+1])
+// 			{
+// 				int temp;
+// 				temp=arr[j];
+// 				arr[j]=arr[j+1];
+// 				arr[j+1]=temp;
+// 			}
+// 		}
+// 	}
+	for(int i=1; i<sizeof(arr)/sizeof(arr[0]); ++i){
+		for(int j=0;j<length-i;++j){
+			if(arr[j]>arr[j+1]){
+// 				int temp;
+// 				temp=arr[j];
+// 				arr[j]=arr[j+1];
+// 				arr[j+1]=temp;
+				swap(&arr[j], &arr[j+1]);
```

Comment:
There is no Built in swap function in C you have to define it

Reply:
Yeah
I forgot to add it